File: Credit Card Mask.js
// return masked string
function maskify(cc) {  //this is the same idea as with the python method, no changes...
  if (cc.length <= 4){
    return cc;
  } else {
    var ok = cc.split("")
    for(var i = 0; i < ok.length - 4; i++){
      ok[i] = "#";
    }
    return ok.join("")
    }
  }

File: Credit Card Mask.py
# return masked string
def maskify(cc):
    if(len(cc) <= 4): #returns the names input if < 4 characters
        return cc
    else:
        new = list(cc)
        for i in range(0, len(new) - 4): #convert to list and change all(except last 4) characters to #
            new[i] = "#"
        new = "".join(new) #convert list to string with no space
        return new
    pass
